dual_net.py

```diff
@@ -265,16 +265,16 @@ def eval_metrics_host_call_fn(policy_output, value_output, pi_tensor, policy_cos
             'value_confidence': tf.metrics.mean(tf.abs(value_output)),
         }
 
+        if est_mode == tf.estimator.ModeKeys.EVAL:
```

Comment:
I wonder if moving this will fix the weird bumps on the summaries.
Previously we might have been updating the summary scalars but not writing them to the file or something.